dev/breeze/src/airflow_breeze/commands/testing_commands.py

```diff
@@ -268,43 +261,12 @@ def run_tests_in_parallel(
     parallel_test_types_list: list[str],
     extra_pytest_args: tuple,
     db_reset: bool,
-    full_tests_needed: bool,
     test_timeout: int,
     include_success_outputs: bool,
     debug_resources: bool,
     parallelism: int,
     skip_cleanup: bool,
 ) -> None:
-    import psutil
```

Comment:
We can now remote this whole part - each test is "smaller" so the memory issues we experienced in the past should be far less frequent.